ansible/server-state-playbooks/nightshade-web/playbook.yml

```diff
@@ -42,18 +23,35 @@
       lvm_lvfilesystem: "{{ provision_root_filesystem }}"
 
     # OMERO.web configuration in host_vars in different repository
```

Comment:
As I became more confident with the Ansible, I realised I could define the vars in the playbook, and keep as much as possible public, only pulling the individual private variables from private config. Initially I thought I'd have to keep the whole `omero_web_config_set` together as one. `ome-dundeeomero` represents my having learned that lesson. A refactor of the ns-web playbook would be nice, in this style.